test/distribution/test_distribution_inference.py

```diff
@@ -253,6 +262,39 @@ def test_multivariate_gaussian() -> None:
     ), f"Sigma did not match: sigma = {Sigma}, sigma_hat = {Sigma_hat}"
 
 
+@pytest.mark.timeout(30)
+def test_dirichlet() -> None:
+    num_samples = 2000
+    dim = 3
+
+    alpha = np.random.gamma(shape=1, size=dim) + 0.5
+
+    distr = Dirichlet(alpha=mx.nd.array(alpha))
+    cov = distr.variance.asnumpy()
+
+    samples = distr.sample(num_samples)
+
+    alpha_hat = maximum_likelihood_estimate_sgd(
+        DirichletOutput(dim=dim),
+        samples,
+        init_biases=None,
+        hybridize=False,
```

Comment:
Does this not work with `hybridize=True`? If it does work, then you can add `hybridize` to the test arguments and try both `False` and `True`.

Reply:
This would also be nice for multivariate Gaussian ones